src/python/nightlyBench.py

```diff
@@ -1840,6 +1870,7 @@ def writeIndexHTML(searchChartData, days):
   writeOneLine(w, done, 'AndHighMedDayTaxoFacets', '+high-freq +medium-freq +dayOfYear taxo facets')
   writeOneLine(w, done, 'BrowseRandomLabelTaxoFacets', 'Random labels chosen from each doc')
   writeOneLine(w, done, 'BrowseRandomLabelSSDVFacets', 'Random labels chosen from each doc (doc values)')
+  w('<br>&nbsp;&nbsp;&nbsp;&nbsp;<a href="nad_facet_benchmarks.html.html">NAD high cardinality faceting</a>')
```

Comment:
Hmm, I have to wonder why not use `writeOneLine` here like the rest of them?

Reply:
From what I discerned, `writeOneLine` is used with the tests configured via the tasks files. The tests in the "Others" section do not use `writeOneLine` as well (see [here](https://github.com/mikemccand/luceneutil/blob/master/src/python/nightlyBench.py#L1861)). I did realize I wrote `.html` twice so I'll fix that though :)